R/init.R

```diff
@@ -76,6 +76,9 @@ copy_assets <- function(pkg = ".") {
   # Copy assets from directory
   if (!is.null(template$assets)) {
     copy_asset_dir(pkg, template$assets)
+  } else {
+    # default directory
+    copy_asset_dir(pkg, file.path("pkgdown", "assets"))
```

Comment:
Should we always copy these files? i.e. so you can combine assets from a template package and from the current package?

Reply:
that's currently the case i.e. with this PR you _can_ combine them which I think is good?

Reply:
But this line is only run if `template$assets` is NULL?